bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java

```diff
@@ -178,6 +181,35 @@ protected void onLayout(boolean changed, int left, int top, int right, int botto
         this.contentClipRect.set(0, 0, getWidth(), bottomClip);
     }
 
+    @Override
+    public boolean onKeyPreIme(int keyCode, KeyEvent event) {
```

Comment:
I will if I have time, I feel like looking at DrawerLayout would be better than AutoCompleteTextView as it's most similar to this view.